class_teacher.rb

```diff
@@ -0,0 +1,21 @@
+require_relative 'class_person'
+
+class Teacher < Person
+  attr_reader :specialization
+
+  def initialize(id, age, specialization, name = 'Unknown', parent_permission: true)
+    super(id, age, name: name, parent_permission: parent_permission)
```

Comment:
- [ ]  Kindly remove the `id` parameter from the `initialize` and `super` declarations since it will be randomly generated within the `Person class.` :+1: